### ISSUE TITLE
Unescaped left brace in regex is deprecated, passed through in regex

### ISSUE THREAD

**Issue by jbodhorn** [baedert/corebird#454]
I'm new to Linux, Arch especially, today is day two with arch... Also, I've only compiled a few programs from source myself during the past couple years I've been attempting to learn Linux. That said I don't know exactly what info is needed to fix this, if it's arch specific, if it's just me, or if it is actually something wrong. 

If any logs or other info is needed please let me know. If it's command based to find the info you'll probably have to give me the command

pastebin from arch aur build attempt:

http://pastebin.com/bzSLQS8b

pastebin for attempt to build from the github sources:

http://pastebin.com/UWQbVVB1

**Comment by baedert** [comment]
The title has nothing to do with the actual problem here.  The first second pastebin is completely fine,
the first one is fine except for the part at the very bottom where you type your password and it's just aborting with "WARNING: Failed to install built package(s)".

**Comment by jbodhorn** [comment]
I never did make and make install after I saw the deprecated errors when trying to build from source. I have so little experience in building from source I thought it wasn't gonna work and didn't bother. I figured that since the aur failed to install and had those deprecated errors(is it an error?) that the same thing was going to happen. 

I built from your source after reading your reply and it's running, sorry to be a bother, thank you for your quick reply. Should I delete or edit my post in someway as to not confuse people?
